app/models/post.rb

```diff
@@ -14,13 +14,13 @@ class Post < ApplicationRecord
 
   after_save :update_post_counter
 
-  private
+  # private
 
-  def update_post_counter
-    author.increment!(:posts_counter)
-  end
+  # def update_post_counter
```

Comment:
- [ ] Great job so far. Please stop commenting out your methods in the models as it will only make your work incomplete
- [ ] kindly check if you have pushed all the changes concerning the unit test. Like the previous reviewer noted, this milestone requires that you add unit tests for User, Post, Comment, and Like model. Any custom methods in the models should have a corresponding unit test. i can't find your test.
Example:

```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```



Any custom methods in the models should have a corresponding unit test.

Reply:
@tufoinnkuo10 The methods I commented out are not required until next week's submissions. If you check the project requirements for this milestone you will notice that those methods were not required

Reply:
@Dadadon please add some unit test for the models or push the code if you forgot to do so